IDE/MonoDevelop/MonoDevelop.MonoGame/templates/MonoGameProject.xpt.xml

```diff
@@ -28,6 +28,7 @@
 			<Files>
 				<File name="Game1.cs" src="Common/Game1.cs" />
 				<File name="Program.cs" src="Common/Program.cs" />
+				<File name="app.manifest" src="Common/app.manifest" />
```

Comment:
I don't think this is required as this is the Desktop GL project. Unless this manifest makes a difference on windows with OpenGL. (I'm not sure it does)